botx/models/chats.py

```diff
@@ -28,3 +29,47 @@ class ChatFromSearch(BotXBaseModel):
 
     #: users in chat.
     members: List[UserFromChatSearch]
+
+    #: creation datetime of chat.
+    inserted_at: datetime
+
+
+class BotChat(BotXBaseModel):
+    """Chat from list."""
+
+    #: name of chat.
+    name: str
+
+    #: description of chat
+    description: Optional[str]
+
+    #: type of chat.
+    chat_type: ChatTypes
+
+    #: ID of chat.
+    group_chat_id: UUID
+
+    #: users in chat.
+    members: List[UUID]
+
+    #: creation datetime of chat.
+    inserted_at: datetime
+
+    #: update datetime of chat.
+    updated_at: datetime
+
+
+class BotChatList(BotXBaseModel):
+    """List of chats."""
```

Comment:
List of chats from bot's chat list request

Reply:
Bot's chat list response model?

Reply:
maybe...

Reply:
Model to `chats/list` response?